reflex/app.py

```diff
@@ -860,6 +868,10 @@ async def upload_file(files: List[UploadFile]):
         for file in files:
             assert file.filename is not None
             file.filename = file.filename.split(":")[-1]
+
+        if app.state_manager is None:
+            raise ValueError("The state manager has not been initialized.")
```

Comment:
isn't this what `validate_state_manager` is for? at any rate, see comment above.